products/green_fast_track/models/intermediate/flags/_flags_models.yml

```diff
@@ -8,13 +8,19 @@ models:
         data_type: string
         tests:
           - not_null
+      - name: flag_id_field_name
```

Comment:
Clarification: it seems like we're redefining what a flag is in this PR - instead of a flag being a unique combo of variable and lot, it maps 1-to-1 with survey questions.

Is Jack using the term "flag"? Or is he really thinking more just about survey questions, and "flag" is more our terminology?

If it's the former, this change makes sense. But if it's the latter, I'd sort of like to keep our definition of flag, since we have an identifier for the columns - "survey questions" (or answers to them, really). If we make flag have the same meaning, we don't have a term for unique combo of variable and bbl anymore.

Reply:
Or maybe this column should more just be "flag_type" or something like that since we do have 1 row per our original "flag" in this table still

Reply:
Jack is using the term "flag" to mean the binary answer to a single survey question

it was hard for me to name `flag_field_name` and `flag_id_field_name` in the new `question_flags.csv`:
- "field name" really seemed worth it because these are all for Jack to choose and alias columns
- each flag has a _value_ (Yes/No) and _IDs_ of geometries that resulted in the value, so I ended up with `flag` and `flag_id`. maybe `flag_ids` would be a little better though
- was tempted to lean more into `survey_question` or the `survey_question_id` values that Jack made for the app, but he cautioned that those IDs could change even if the essence of the question doesn't. so seemed good to just declare `flag_field_name` values that are 1-to-1 with survey questions